File: patterns/pattern_6.cpp
#include<iostream>
using namespace std;
int main()
{
	int n;
	cout <<"Enter a range: ";
	cin >>n;
	for(int i=1; i<=n; i++)
	{
		for(int j=n; j>=i; j--)
		{
			cout <<" ";
		}
		for(int k=2; k<=2*i; k++)
		{
			cout <<"*";
		}
		cout <<endl;
	}
	return 0;
}

File: patterns/Pattern6.java
import java.util.*;

class Pattern6{

	public static void main(String args[]){
		int n;
		Scanner sc= new Scanner(System.in);
		System.out.print("Enter a range: ");
		n=sc.nextInt();
		for(int i=1; i<=n; i++)
		{
			for(int j=n; j>=i; j--)
			{
				System.out.print(" ");
			}
			for(int k=2; k<=2*i; k++)
			{
				System.out.print("*");
			}
			 System.out.println();
		}
	}
}